bot/handlers/connect_button.py

```diff
@@ -10,14 +12,29 @@
 async def connect_button(message: Message):
     user_id = message.from_user.id
     try:
-        key = await vpn_client.get_client_key(user_id)
-        if key:
-            text = f"<pre>{key}</pre>"
+        user = await get_user_by_telegram_id(user_id)
+        if user.expires_at and user.expires_at <= datetime.now(
+            timezone.utc
+        ):  # 1) действующий пользователь с истекшей подпиской
```

Comment:
```suggestion
        is_user_with_expired_subscription = user.expires_at and user.expires_at <= datetime.now(timezone.utc)
        if is_user_with_expired_subscription:
```
Лучше завести переменную с внятным названием, чем добавлять комментарий. Не везде это возможно, но тут можно.

Reply:
ok